contents/founders/cracked-manifesto.md

```diff
@@ -0,0 +1,87 @@
+---
+date: 2024-04-17
+title: Why you need to hire cracked engineers
+author:
+  - charles-cook
+featuredImage: https://res.cloudinary.com/dmukukwp6/image/upload/v1713344206/posthog.com/contents/blog/cracked_engineer_blog.jpg
+tags:
+  - Culture
+  - People
+crosspost:
+  - Blog      
+---
+
+We’ve become a bit obsessed with the notion of hiring cracked engineers at PostHog. When we say cracked, we mean things like:
+
+- Taking extreme ownership of ideas and driving them to completion, even if others disagree with you. Some people call this founder mentality. Better still, you bring others on the journey with you.
+ 
+- Unwavering optimism in the face of change or new ideas. This doesn’t mean you have to agree with the change, but adopting a ‘yes and’ mentality helps new ideas get off the ground that might otherwise die in a committee somewhere. This is especially true for ideas that come from non-senior or new people.
+ 
+- People leave a meeting or conversation with you more excited/energized than when they started. You build people and their ideas up, and are generally perceived as a very helpful person.
+ 
+- Behaving in a completely authentic way. You don’t play politics – your work does the talking. For example, if you do marketing, talk like an actual human being. This doesn’t work in most large organizations, sadly.
+
+- You apply yourself to the craft of your discipline with a quasi-religious fervor. The joy comes from the craft itself, not creating shareholder value.  
```

Comment:
```suggestion
- **Taking extreme ownership of ideas** and driving them to completion, even if others disagree with you. Some people call this founder mentality. Better still, you bring others on the journey with you.
 
- **Unwavering optimism** in the face of change or new ideas. This doesn’t mean you have to agree with the change, but adopting a ‘yes and’ mentality helps new ideas get off the ground that might otherwise die in a committee somewhere. This is especially true for ideas that come from non-senior or new people.
 
- **You make people feel more excited and energized** after a meeting or conversation with you. You build people and their ideas up, and are generally perceived as a very helpful person.
 
- **Behaving in a completely authentic way**. You don’t play politics – your work does the talking. For example, if you do marketing, talk like an actual human being. This doesn’t work in most large organizations, sadly.

- **You apply yourself to the craft** of your discipline with a quasi-religious fervor. The joy comes from the craft itself, not creating shareholder value.  
```